src/xcode/ENA/ENA/Source/Workers/SQLiteKeyValueStore.swift

```diff
@@ -59,12 +67,29 @@ class SQLiteKeyValueStore {
 			return
 		}
 		let sqlStmt = """
+		PRAGMA auto_vacuum=2;
+
 		CREATE TABLE IF NOT EXISTS kv (
 			key TEXT UNIQUE,
 			value BLOB
 		);
 		"""
-		db.executeStatements(sqlStmt)
+		if !db.executeStatements(sqlStmt) {
+			removeDatabase()
+		}
```

Comment:
Just want to point to this change to get some Feedback on it. When the database can't be accessed after the setup process with setting the encryption key. We have to assume that the Database file is not in a valid format or damaged. Therefore we remove it and reset the store.